ansible/roles/test/tasks/continuous_link_flap.yml

```diff
@@ -0,0 +1,216 @@
+      register: orch_cpu
+      until: "{{orch_cpu.stdout|int}} < {{orch_cpu_threshold}}"
+      retries: 100
+      delay: 2
+
+    - debug: msg="First Iteration flap all interfaces one by one on DUT"
+
+    - name: Gathering lab graph facts about the device
+      conn_graph_facts: host={{ inventory_hostname }}
+      connection: local
+      tags: always
+
+    - debug: msg="{{ device_conn }}"
+
+    - name: include continuous_link_flap_helper.yml
+      include: continuous_link_flap/continuous_link_flap_helper.yml interface="{{item}}"
+      with_items: "{{ device_conn.keys() }}"
+
+    - debug: msg="Second Iteration flap all interfaces one by one on DUT"
+
+    - name: Gathering lab graph facts about the device
+      conn_graph_facts: host={{ inventory_hostname }}
+      connection: local
+      tags: always
+
+    - name: include continuous_link_flap_helper.yml
+      include: continuous_link_flap/continuous_link_flap_helper.yml interface="{{item}}"
+      with_items: "{{ device_conn.keys() }}"
+
+    - debug: msg="Third Iteration flap all interfaces one by one on DUT"
+
+    - name: Gathering lab graph facts about the device
+      conn_graph_facts: host={{ inventory_hostname }}
+      connection: local
+      tags: always
+
+    - name: include continuous_link_flap_helper.yml
+      include: continuous_link_flap/continuous_link_flap_helper.yml interface="{{item}}"
+      with_items: "{{ device_conn.keys() }}"
+
+    - debug: msg="First Iteration flap all interfaces one by one on Peer Device"
+
+    - name: Gathering lab graph facts about the device
+      conn_graph_facts: host={{ inventory_hostname }}
+      connection: local
+      tags: always
+
+    - debug: msg="{{ device_conn }}"
+
+    - set_fact:
+        neighbors: "{{device_conn}}"
+
+    - name: include continuous_peer_link_flap_helper.yml
+      include: continuous_link_flap/continuous_peer_link_flap_helper.yml interface="{{item}}"
+      with_items: "{{ device_conn.keys() }}"
+
+    - debug: msg="Second Iteration flap all interfaces one by one on Peer Device"
+
+    - name: Gathering lab graph facts about the device
+      conn_graph_facts: host={{ inventory_hostname }}
+      connection: local
+      tags: always
+
+    - set_fact:
+        neighbors: "{{device_conn}}"
+
+    - name: include continuous_peer_link_flap_helper.yml
+      include: continuous_link_flap/continuous_peer_link_flap_helper.yml interface="{{item}}"
+      with_items: "{{ device_conn.keys() }}"
+
+    - debug: msg="Third Iteration flap all interfaces one by one on Peer Device"
+
+    - name: Gathering lab graph facts about the device
+      conn_graph_facts: host={{ inventory_hostname }}
+      connection: local
+      tags: always
+
+    - set_fact:
+        neighbors: "{{device_conn}}"
+
+    - name: include continuous_peer_link_flap_helper.yml
+      include: continuous_link_flap/continuous_peer_link_flap_helper.yml interface="{{item}}"
```

Comment:
>item [](start = 86, length = 4)

I tried run your test case and hit an error here.
```
TASK [test : debug] ************************************************************
Saturday 27 April 2019  01:55:47 +0000 (0:00:00.236)       0:40:37.453 ********
ok: [sonic] => {
    "msg": "Third Iteration flap all interfaces one by one on Peer Device"
}

TASK [test : Gathering lab graph facts about the device] ***********************
Saturday 27 April 2019  01:55:48 +0000 (0:00:00.430)       0:40:37.884 ********
ok: [sonic]

TASK [test : set_fact] *********************************************************
Saturday 27 April 2019  01:55:49 +0000 (0:00:00.794)       0:40:38.678 ********
ok: [sonic]

TASK [test : include continuous_peer_link_flap_helper.yml] *********************
Saturday 27 April 2019  01:55:49 +0000 (0:00:00.434)       0:40:39.113 ********
included: ansible/roles/test/tasks/continuous_link_flap/continuous_peer_link_flap_helper.yml for sonic

TASK [test : debug] ************************************************************
Saturday 27 April 2019  01:56:00 +0000 (0:00:10.718)       0:40:49.831 ********
fatal: [sonic]: FAILED! => {"failed": true, "msg": "ERROR! 'item' is undefined"}
```

Reply:
Seems,  Ports are down in your testbed after 3rd iteration. So the item variable is not set.
I never observe this issue in my lab. So I can not resolve it. Please try to rerun and It should pass. Thanks.